content/pt/docs/concepts/scheduling-eviction/pod-overhead.md

```diff
@@ -13,37 +12,37 @@ weight: 50
 {{< feature-state for_k8s_version="v1.18" state="beta" >}}
 
 Quando executa um Pod num nó, o próprio Pod usa uma quantidade de recursos do sistema. Estes
-recursos são adicionais aos recursos necessários para executar o(s) _container(s)_ dentro do Pod.
+recursos são adicionais aos recursos necessários para executar o(s) contêiner(s) dentro do Pod.
 Sobrecarga de Pod, do inglês _Pod Overhead_, é uma funcionalidade que serve para contabilizar os recursos consumidos pela
-infraestrutura do Pod para além das solicitações e limites do _container_.
+infraestrutura do Pod para além das solicitações e limites do contêiner.
 
 
 
 
 
 <!-- body -->
 
-No Kubernetes, a sobrecarga de _Pods_ é definido no tempo de
+No Kubernetes, a sobrecarga de Pods é definido no tempo de
 [admissão](/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks)
 de acordo com a sobrecarga associada à
-[RuntimeClass](/docs/concepts/containers/runtime-class/) do _Pod_.
+[RuntimeClass](/docs/concepts/containers/runtime-class/) do Pod.
 
 Quando é ativada a Sobrecarga de Pod, a sobrecarga é considerada adicionalmente à soma das
-solicitações de recursos do _container_ ao agendar um Pod. Semelhantemente, o _kubelet_
+solicitações de recursos do contêiner ao agendar um Pod. Semelhantemente, o _kubelet_
 incluirá a sobrecarga do Pod ao dimensionar o cgroup do Pod e ao
```

Comment:
the term "despejo" here confused me. The scenario is: I need to drain a node, the pod needs to be evicted and re-scheduled to another node. 

Maybe we could use "a classificação de prioridade de migração do Pod em caso de _drain_ do Node"?

Reply:
makes more sense to me, the "despejo" word exists in the original translation, I liked your suggestion!